### PR TITLE
Create ui-c8y-1021-0-0-info-gauge-widget-should-support-device-type-dashboards-7154.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by reey

Ticket: [MTM-61069](https://cumulocity.atlassian.net/browse/MTM-61069)
Original PR: [7154](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7154)

[MTM-61069]: https://cumulocity.atlassian.net/browse/MTM-61069?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-info-gauge-widget-should-support-device-type-dashboards-7154.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: info gauge widget should support device type dashboards (#7154)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-info-gauge-widget-should-support-device-type-dashboards-7154.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Info gauge widget now supports device type dashboards
```

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-info-gauge-widget-should-support-device-type-dashboards-7154.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-61069
+version: 1021.0.0
+---
+info gauge widget should support device type dashboards (#7154)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-info-gauge-widget-should-support-device-type-dashboards-7154.md:17]: The Claude AI suggested to use the following body:
```suggestion
Previously, the info gauge widget did not support device type dashboards, limiting its usability for users who rely on these dashboards. With this change, the info gauge widget has been updated to fully support device type dashboards. This enhancement allows users to effectively utilize the info gauge widget within their device type dashboard setups, providing them with a more comprehensive and flexible monitoring experience.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-info-gauge-widget-should-support-device-type-dashboards-7154.md (hunk 2 above)
Comment (rahulc8y) [reply content/change-logs/application-enablement/ui-c8y-1021-0-0-info-gauge-widget-should-support-device-type-dashboards-7154.md:17]:
```suggestion
Previously, the info gauge widget did not support device type dashboards, which limited its usability. With this change, the info gauge widget has been updated to fully support device type dashboards. This update enables users to add and configure info gauge widgets within device type dashboards, providing greater flexibility in dashboard configurations.
```

**Comment by reey** [comment]
closing as already covered in: https://github.com/Cumulocity-IoT/c8y-docs/pull/2486

**Pull request closed without merging**